### ISSUE TITLE
No implementation found for method storeSdkInfo on channel qonversion_flutter_sdk error

### ISSUE THREAD

**Issue by kobeumut** [qonversion/flutter-sdk#75]
```   qonversion_flutter: ^3.0.1 ```

I want to switch from revenuecat to qonversion but when I run my app I have an error below line.
```
await Qonversion.launch('****-vY7N7', isObserveMode: false);
```
```
E/flutter: [ERROR:flutter/lib/ui/ui_dart_state.cc(199)] Unhandled Exception: MissingPluginException(No implementation found for method storeSdkInfo on channel qonversion_flutter_sdk)
    #0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:156:7)
    <asynchronous suspension>
E/flutter: [ERROR:flutter/lib/ui/ui_dart_state.cc(199)] Unhandled Exception: MissingPluginException(No implementation found for method launch on channel qonversion_flutter_sdk)
    #0      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:156:7)
    <asynchronous suspension>
    #1      Qonversion.launch (package:qonversion_flutter/src/qonversion.dart:49:23)
    <asynchronous suspension>
    #2      Initialization.initPlatformState (package:flutter_module/app/controllers/initialization.dart:281:38)
    <asynchronous suspension>
```

After that I try to pub get, I have an error.
```
❯ flutter pub get
Running "flutter pub get" in flutter_module...                     928ms
The plugin `qonversion_flutter` is built using an older version of the Android plugin API which assumes that it's running in a full-Flutter environment. It may have undefined behaviors when Flutter is integrated into
an existing app as a module.
The plugin can be updated to the v2 Android Plugin APIs by following https://flutter.dev/go/android-plugin-migration.
```

By the way I tried flutter clean, and also I am using FlutterView on Android.

**Comment by Maria-Bordunova** [comment]
Hi, @kobeumut!
Thank you for your report.
I need a bit more time to find out with your issue. I'll come back soon

**Comment by Maria-Bordunova** [comment]
@kobeumut 
I can confirm this problem. As soon as an app uses the Android v2 embedding, qonversion_flutter plugin stops functioning (like in your case). It happens because qonversion android plugin uses v1 embedding, which doesn't provide backward compatibility for v2 embedding.
We've made this task a priority. It's hard to predict how long it will take. But we're working on it right now. Let us fix it. And then you'll be able to move to Qonversion and test our functionality :handshake:

**Comment by Maria-Bordunova** [comment]
@kobeumut , afternoon :slightly_smiling_face:
We've published new release version 3.2.0. It supports now Android v2 embedding.
Please check it out, it should fix your problem 🔨 .

**Comment by kobeumut** [comment]
sorry, I had no time, but finally, I tried and it works. Thank you.
